### DISCUSSION TITLE
post/kakaopaysec-mongodb-cdc/

### DISCUSSION THREAD

**Discussion by giscus** [kakaopay/techlog-comments#59] [Announcements]
# post/kakaopaysec-mongodb-cdc/

Oracle에서 MOngoDB로의 초기 데이터 이관 및 CDC Pipeline 구축 경험을 공유합니다.

https://tech.kakaopay.com/post/kakaopaysec-mongodb-cdc/

**Comment by ghkdqhrbals** [comment]
좋은 글 감사합니다!
